src/Symfony/Component/HttpClient/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+* added `MockHttpClient::setResponseFactory()` method to be able to set response factory after client creating
```

Comment:
missing space at start of line
"added" should be replaced by "Add"

Reply:
Fixed